### PR TITLE
implement commonly used functionality

### PR DESCRIPTION
**Pull request by robertkowalski**
this implements commonly used functionality to create dazaar cores

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+      path.join(storageDir, 'core', name),
+      keyPair.publicKey,
+      { secretKey: keyPair.secretKey }
+    )
+
+    const fready = promisify(feed.ready).bind(feed)
+    await fready()
+
+    const seller = market.sell(feed, {
+      validate (remoteKey, done) {
```
Comment (mafintosh) [inline index.js:34]: `return done(null, { type: 'free' })` - then you can drop the payment module which is super heavy :)

File: package.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@bitfinex/create-free-dazaar-core",
```
Comment (mafintosh) [inline package.json:2]: @dazaar/create-free-core perhaps? we own that org

File: index.js (hunk 1 above)
Comment (robertkowalski) [reply index.js:34]: nice!!! done!

File: package.json (hunk 2 above)
Comment (robertkowalski) [reply package.json:2]: lets keep it here for now, got this nice and shiny repo :)

**Pull request merged**